app/views/admin/index.html.haml

```diff
@@ -1,14 +1,15 @@
 =h1 "Administration"
 %ul
+  %li= link_to "Applications de l’API", admin_applications_path
```

Comment:
I was surprised to have seen that at the top of the list, is it often used ?

Instead of having only one big list, maybe it could be useful to split the links by theme.

Reply:
I just lexically reordered.

The only link frequently used is "Derniers comptes", and so frequently that I don't use the index anymore, I just asked "admin/comptes" or "comptes" to my browser... So I thought why not put them into lexical order instead of random order.